### DISCUSSION TITLE
Asking for Phone Number or Bot Token

### DISCUSSION THREAD

**Discussion by ImArghya7** [Dineshkarthik/telegram_media_downloader#178] [Q&A] [answered]
<img width="920" alt="raycast-untitled" src="https://user-images.githubusercontent.com/22514844/155946087-1f4a4070-170a-4426-b0c9-7a6538e17d9b.png">

What should I do next?

PS: 1st-time user, really rookie!

**Comment by Dineshkarthik** [comment, marked as answer]
Hi,

Enter your phone number associated with your Telegram account. And then you will receive an OTP for this login in your Telegram which you need to enter here.

**Reply by ImArghya7** [reply]
Thank you very much for the help.
